packages/backend/src/file/file.service.spec.ts

```
@@ -56,3 +58,228 @@ describe('FileService.upload image support', () => {
+    );
+    expect(id).toMatch(/\.csv$/);
+    expect(send).toHaveBeenCalledTimes(1);
+  });
+
+  it('does not let that fallback admit a binary .xls', async () => {
+    const svc = makeService();
+    withFakeS3(svc);
+    // Same unreliable type, but an extension nothing here can read. Only the
+    // extension is honoured, never the browser's claim.
+    await expect(
+      svc.upload(Buffer.from('x'), 'application/vnd.ms-excel', 'sales.xls', { category: 'data', workspaceId: 'ws1' }),
```

ESLint

[error] Replace `·category:·'data',·workspaceId:·'ws1'` with `⏎········category:·'data',⏎········workspaceId:·'ws1',⏎·····`

(prettier/prettier)